### DISCUSSION TITLE
Missing pages

### DISCUSSION THREAD

**Discussion by PixLSteam** [pokeclicker/pokeclicker-wiki#4] [Ideas]
Quite a few pages are still missing, here are some of them:

- [ ] Farm Hands
- [x] Routes
- [x] Towns
- [ ] Currencies?
- [ ] Underground
- [ ] Roaming Pokémon

Feel free to reply with other missing pages. Please notify me if any of these are done or not wanted so I can update the list.

**Comment by RedSparr0w** [comment]
✅ Routes completed

Gonna work on adding towns next I think

**Comment by RedSparr0w** [comment]
✅ Added basic towns

Could probably use a little work, but will do for now.
